README.md

```diff
@@ -1,16 +1,12 @@
 # infrastructure
 
-Repo to track code + issues relating to IPFS infrastructure for users + devs.
-This includes:
+This repository contains the technical infrastructure of the IPFS community.
 
-Computing:
-- Gateway nodes
-- Bootstrap nodes
-- CI servers (jenkins, travis, etc)
+- Gateway (http://gateway.ipfs.io) and bootstrap nodes, in `solarnet/`
+- Seeding of $important objects, in `seeding/`
 
-Communications:
-- Github repos
-- IRC Channel
-- Mailing Lists
```

Comment:
i want to keep (and expand) reference to these things here. the IPFS infrastructure is more than just the programs we run. IRC, mailing list, github, etc are all important to account.